pom.xml

```diff
@@ -2426,10 +2427,25 @@
       </dependency>
       <dependency>
         <groupId>org.apache.httpcomponents</groupId>
+        <artifactId>httpclient</artifactId>
+        <version>${version.org.apache.httpcomponents.httpclient}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.httpcomponents</groupId>
+        <artifactId>fluent-hc</artifactId>
+        <version>${version.org.apache.httpcomponents.httpclient}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpmime</artifactId>
         <version>${version.org.apache.httpcomponents.httpclient}</version>
       </dependency>
       <dependency>
+        <groupId>org.apache.httpcomponents</groupId>
+        <artifactId>httpcore</artifactId>
+        <version>${version.org.apache.httpcomponents.httpcore}</version>
+      </dependency>
+      <dependency>
```

Comment:
Is this needed? I believe we only need to override the property?

Reply:
It seems if you only override the property but not explicitly the artifact the artifact coming from jboss-ip-bom has the version of ip-bom.
httpcore was not upgraded to the version we did an override in kie-parent

Reply:
This was the case for 6.x, but for 7.x the property override is enough. Just verified that locally with effective pom.